cmd/olm/manager.go

```diff
@@ -13,7 +13,7 @@ import (
 var log = ctrl.Log.WithName("manager")
 
 func Manager(ctx context.Context, debug bool) (ctrl.Manager, error) {
-	ctrl.SetLogger(zap.Logger(debug))
+	ctrl.SetLogger(zap.New(zap.UseDevMode(debug)))
```

Comment:
Do we want debug mode?

Reply:
I believe so, the value is based on a flag provided via at runtime.